corehq/apps/domain/tests/test_two_factor_check.py

```diff
@@ -28,46 +30,39 @@ def create_request(cls, request_url):
         request.couch_user = CouchUser()
         return request
 
-    @flag_enabled('TWO_FACTOR_SUPERUSER_ROLLOUT')
-    def test_two_factor_required_with_feature_flag(self):
+    def test_two_factor_required_for_superuser(self):
         view_func = 'dummy_view_func'
         request = self.request
-        two_factor_required_bool = _two_factor_required(view_func, self.domain, request.couch_user)
-        self.assertTrue(two_factor_required_bool)
+        self.assertFalse(_two_factor_required(view_func, self.domain, request.couch_user))
 
-    def test_two_factor_required_without_feature_flag(self):
-        view_func = 'dummy_view_func'
-        request = self.request
-        two_factor_required_bool = _two_factor_required(view_func, self.domain,
-                                                        request.couch_user)
-        self.assertFalse(two_factor_required_bool)
+        request.couch_user.is_superuser = True
+        self.assertTrue(_two_factor_required(view_func, self.domain, request.couch_user))
```

Comment:
i think this test assumes `settings.ENFORCE_TWO_FACTOR_FOR_SUPERUSERS == True`. is that by design? seems you could use `override_settings` in the positive case as well if we wanted people to be able to disable the setting locally but still run this test.

Reply:
Yes, I did make it `True` by default. I'm a bit surprised by the test failures since I don't think tests should use superuser's unless specifically testing something related.